app/code/community/CustomGento/ConfigurableTierPrices/Helper/Data.php

```diff
@@ -41,15 +39,15 @@ public function isProductInDisabledCategory(Mage_Catalog_Model_Product $product)
 
     public function isExtensionDisabledForProduct(Mage_Catalog_Model_Product $product)
     {
-        // get the product attribute directly, because it may not be loaded
+        // Get the product attribute directly, because it may not be loaded
```

Comment:
Please do not change such subtle things in a PR. It makes it harder to merge (and I prefer starting with lower case letters btw).